app/i18n/en/index.ts

```diff
@@ -874,9 +876,11 @@ const en: BaseTranslation = {
     joinTheCommunity: "Join the community",
     whatsapp: "WhatsApp",
     email: "Email",
+    enjoyingApp: "Enjoying the app?",
     statusPage: "Status Page",
     telegram: "Telegram",
     mattermost: "Mattermost",
+    thankyouText: "Thankyou for the feedback, would you like to suggest an improvement?",
```

Comment:
```suggestion
    thankyouText: "Thank you for the feedback, would you like to suggest an improvement?",
```

Reply:
You marked this as resolved, but it is still present. Do you have changes that need to be pushed?

Reply:
Yes, not pushed yet